CreeDictionary/API/models.py

```diff
@@ -197,20 +197,29 @@ def serialize(self) -> SerializedWordform:
             definition.serialize() for definition in self.definitions.all()
         ]
         result["lemma_url"] = self.get_absolute_url()
-        result["wordclass_help"] = self.get_user_friendly_wordclass_help_for_cree()
+
+        # Displayed in the word class/inflection help:
+        result["inflectional_category_plain_english"] = LABELS.english.get(
+            self.inflectional_category
+        )
+        result["inflectional_category_linguistic"] = LABELS.linguistic_long.get(
+            self.inflectional_category
+        )
+        result["wordclass_emoji"] = self.get_emoji_for_cree_wordclass()
 
         return result
 
-    def get_user_friendly_wordclass_help_for_cree(self) -> Optional[str]:
+    def get_emoji_for_cree_wordclass(self) -> Optional[str]:
         """
-        Attempts to get a description, e.g., "like: wîcihêw" or "like:
-        micisow" for this wordform.
+        Attempts to get an emoji description of the full wordclass.
+        e.g., "👤👵🏽" for "nôhkom"
         """
         maybe_full_word_class = self.full_word_class
```

Comment:
I have some thoughts regarding this `full_word_class` property.

Our Wordform model has a non-standard `pos` field (as part of speech is depreacted), which should be replaced to a `word_class` field.

But if we are switching to a JIT paradigm, the way of calculating `full_word_class` on the fly is a good way to start.

BTW, shouldn't it be just "word_class"? The "full" is unnecessary to me.

Reply:
Sure! I'll rename it to `word_class`!